docs/operator-manual/applicationset/Generators-Git.md

```diff
@@ -339,6 +339,8 @@ The filename can always be accessed using `{{.path.filename}}`.
 
 **Note**: The default behavior of the Git file generator is very greedy. Please see [Git File Generator Globbing](./Generators-Git-File-Globbing.md) for more information.
 
+**Note**: For YAML file you have to omit the initial dot (.) when referring to parameters. E.g.: {{cluster.address}}
```

Comment:
I'm confused by this... if you're using `goTemplate: true`, the go text templating engine will interpret whatever is between the `{{}}`. It will fail for the given example.